schemas/pe.yml

```
@@ -39,9 +36,26 @@
       description: Internal product name of the file, provided at compile-time.
       example: Microsoft® Windows® Operating System
 
-
     - name: company
       level: extended
       type: keyword
       description: Internal company name of the file, provided at compile-time.
       example: Microsoft Corporation
+
+    - name: imphash
+      level: extended
+      type: keyword
+      short: A hash of the imports in a PE file.
+      description: >
+        A hash of the imports in a PE file. An imphash is a sort of fuzzy hash that can
+        be used to fingerprint binaries even after recompilation or other code-level
+        transformations have occurred that might change more traditional hash values.
```

I think I would make this description a little more straightforward. What about:

```suggestion
        A hash of the imports in a PE file. An imphash -- or import hash -- can
        be used to fingerprint binaries even after recompilation or other code-level
        transformations have occurred, which would change more traditional hash values.
```

---

Not married to my suggestion either, I find it a bit repetitive. But I'm just trying to tie up the "sort of" and the part at the end, where a traditional hash actually changes completely, even on the smallest change to any part of the file.